src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -125,6 +129,34 @@ public KubernetesCloud(String name) {
         super(name);
     }
 
+    public String getMaxPodReconnectionAttempts() {
+        return maxPodReconnectionAttempts;
+    }
+
+    @DataBoundSetter
+    public void setMaxPodReconnectionAttempts(String maxPodReconnectionAttempts) {
+        if(maxPodReconnectionAttempts.isEmpty()) {
```

Comment:
better to use `StringUtils.isBlank`

Reply:
changed `maxPodReconnectionAttempts` to `int` so this code is no longer needed.